### ISSUE TITLE
Can we set default permissions to 777 for everything in MAIN?

### ISSUE THREAD

**Issue by nfriedly** [MiyooCFW/buildroot#98]
I just booted up linux to copy over some games directly, and noticed that I didn't have permissions to write to the roms folder. It was easy enough to fix (`sudo chmod -R 777 roms`), but I think we should just make that the default.

**Comment by TriForceX** [comment]
of course! i thought this was done before, but checking commits apparently it was applied on uMTP side

**Comment by TriForceX** [comment]
Apparently theres some perms issues in some folders too using uMTP as someone reported recently (eg: gmenu2x), so must be checked in all methods.

**Comment by Apaczer** [comment]
small reading: https://forums.raspberrypi.com/viewtopic.php?t=369199 || https://askubuntu.com/questions/251206/mount-usb-drive-with-write-permissions-for-everyone-or-specific-user

FAT never respected permissions on drive, and anything printed with `ls -a` is just virtually non-existent (test with BOOT partition and see that even so u have only READ access you can still WRITE files). I believe BTRFS behaves here normally allowing only for root to have RW (was it anytime different for linux ex-drivers to behave opposite with auto-mount?).

Anyway I would add documenting how to allow for other user groups to have RW (666, not necessary 777) permissions on mounted externally drives (like using `pmount` cmd: https://www.linuxquestions.org/questions/linux-hardware-18/creating-an-udev-rule-mounting-all-usb-drive-as-666-a-807325/page2.html) and not relying soly on auto-mount feature.

**Comment by Apaczer** [comment]
FYI looks like `sudo chown -R $USER:$USER /media/$USER/MAIN` do the trick permanently at least on Ubuntu machine.
